src/exodus/tx.cpp

```diff
@@ -37,6 +37,11 @@ using boost::algorithm::token_compress_on;
 
 using namespace exodus;
 
+namespace exodus
+{
+boost::signals2::signal<void (CMPTransaction* const)> NotifyProcessedTransaction;
```

Comment:
This is not right. What it mean is _A pointer to non-const CMPTransaction with argument as const_. It should be `CMPTransaction const *` instead.